nservicebus/operations/performance-counters-slaviolation.include.md

```diff
@@ -0,0 +1,16 @@
+## SLA violation countdown
+
+**Counter Name:** `SLA violation countdown`
+
+**Added in:** Version 3
+
+Acts as a early warning system to inform on the number of seconds left until the SLA for the endpoint is breached. This gives a system-wide counter that can be monitored without putting the SLA into the monitoring software. Just set that alarm to trigger when the counter goes below X, which is the time that the operations team needs to be able to take actions to prevent the SLA from being breached.
```

Comment:
> Just set that alarm to trigger when the counter goes below X, which is the time that the operations team needs to be able to take actions to prevent the SLA from being breached.


Also not very clear to me... how do I set the alarm? where? how do I know how much time ops will need?

I think it'll be easier to understand if we explained how it'll behave, e.g. "the alarm will be triggered if...", "a notification will be issued when the threshold is exceeded...", or something along these lines